src/backend/core/product/models.py

```diff
@@ -166,6 +186,9 @@ class ProductPrice(models.Model):
     update_at = models.DateTimeField(auto_now=True)
     create_at = models.DateTimeField(auto_now_add=True)
 
+    # class Meta:
```

Comment:
Can we delete this if its not needed?

Reply:
Yeah, sure, I'll delete it soon. Sooorry